api/api.go

```diff
@@ -42,6 +62,18 @@ func New() *API {
 		lookupAddr:    net.LookupAddr,
 		lookupCountry: func(ip net.IP) (string, error) { return "", nil },
 		ipFromRequest: ipFromRequest,
+		client: &http.Client{
```

Comment:
If the proxy is speaking HTTP on the requested port it would also give a false positive. I'm not sure it's worth the extra complexity. What do you think?